ykrt/src/mt.rs

```diff
@@ -27,11 +26,11 @@ const PHASE_NUM_BITS: usize = 3;
 const PHASE_TAG: usize = 0b111; // All of the other PHASE_ tags must fit in this.
 const PHASE_COMPILED: usize = 0b000; // Value is a pointer to a chunk of memory containing a
                                      // CompiledTrace<I>.
-const PHASE_TRACING: usize = 0b001; // Value is a pointer to a malloc'd block that allows us to
-                                    // precisely identify when we have reached the same Location.
+const PHASE_TRACING: usize = 0b001; // Value is a pointer to an Arc<u8> representing a thread ID.
 const PHASE_COMPILING: usize = 0b010; // Value is a pointer to a `Box<CompilingTrace<I>>`.
 const PHASE_COUNTING: usize = 0b011; // Value specifies how many times we've seen this Location.
 const PHASE_LOCKED: usize = 0b100; // No associated value.
+const PHASE_WONT_TRACE: usize = 0b101; // No associated value.
```

Comment:
I'll throw in my obligatory bikeshed comment and say: s/WONT/DONT/g ?

Reply:
I feel not strongly one way nor either the other. Though I'd have written that previous sentence "I don't care one way or t'other" myself :p

Reply:
On reflection, I agree. Fixed in https://github.com/softdevteam/yk/pull/187/commits/0ea56e8b8a2a783bf69e9e365587a4c1b1461aa6.